jsdoc/tag-defs/access.js

```diff
@@ -1,13 +1,16 @@
-module.exports = function(accessTagTransform, trimWhitespaceTransform) {
-  var name = 'access'
-  accessTagTransform.addTag(name);
-  accessTagTransform.addValue('private');
-  accessTagTransform.addValue('protected');
-  accessTagTransform.addValue('public');
-
+module.exports = function(createDocMessage) {
+  STANDARD_VALUES = ['private', 'protected', 'public'];
   return {
-    name: name,
-    docProperty: 'access',
-    transforms: [trimWhitespaceTransform, accessTagTransform]
+    name: 'access',
+    transforms: function(doc, tag, value) {
+      var value = value.trim();
+      if (STANDARD_VALUES.indexOf(value) !== -1) {
+        if (doc.tags.getTags(value).length > 0) {
+          throw new Error(createDocMessage('"@access ' + value + '" tag cannot be used with "@' + value + '" tag on the same document', doc));
```

Comment:
The `parseTagsProcessor` already deals with this because this tag has not declared itself as `multi: true`. Compare with `@param`: https://github.com/angular/dgeni-packages/blob/master/jsdoc/tag-defs/param.js#L4